R/fda-table_33.R

```diff
@@ -0,0 +1,62 @@
+# Pre-processing
+
+library(tidyverse)
+library(scda)
+library(tern)
+
+# Safety population
+adsl <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "adsl") %>%
+  dplyr::filter(SAFFL == "Y") %>%
+  dplyr::select(STUDYID, USUBJID, SAFFL, ARM, ACTARM)
```

Comment:
We don't need to select certain variables since extra variables will not be used in the table.
```suggestion
  dplyr::filter(SAFFL == "Y")
```